app/controllers/APIController.scala

```diff
@@ -54,7 +55,15 @@ class APIController @Inject()(etmpService: ETMPService,
   }
   private def returnResponseForAPI(etmpPayload:ETMPPayload):Result = {
     val pointsTotal = etmpPayload.pointsTotal
-    val responseData:APIModel = APIModel(pointsTotal)
+    val noOfEstimatedPenalties = getNumberOfEstimatedPenalties(etmpPayload)
+
+    val responseData:APIModel = APIModel(pointsTotal,noOfEstimatedPenalties)
     Ok(Json.toJson(responseData))
   }
+
+  private def getNumberOfEstimatedPenalties(etmpPayload: ETMPPayload): Int = {
+    val lppEstimatedPenalties:Int = etmpPayload.latePaymentPenalties.map(_.count(_.status==PointStatusEnum.Estimated)).getOrElse(0)
+    val lspEstimatedPenalties:Int = etmpPayload.penaltyPoints.count(_.status==PointStatusEnum.Estimated)
+      lppEstimatedPenalties + lspEstimatedPenalties
+  }
```

Comment:
Could you move this out of the controller and into the ETMPService?

Reply:
I think it would be good to isolate this logic and be able to test it in isolation 😄

Reply:
> I think it would be good to isolate this logic and be able to test it in isolation 😄

👍